bin/check-azurerm-monitor-metric.rb

```diff
@@ -127,12 +127,16 @@ class CheckAzurermMonitorMetric < Sensu::Plugin::Check::CLI
          short: '-f FILTER',
          long: '--filter FILTER'
 
-  option :aggregation,
-         description:  'Aggregation.  This can be Average, Count, Maximum, Minimum, Total',
+  option :request_aggregation,
+         description:  'Used as a parameter to the HTTP request sent to Azure.   This can be Average, Count, Maximum, Minimum, Total',
          short: '-a aggregation',
          long: '--aggregation aggregation',
          default: 'average'
 
+  option :aggregate_results,
+         description: 'Aggregate the result data points to compare against alert conditions.   This can be Average, Count, Maximum, Minimum, Total',
```

Comment:
We should probably lowercase them since as far as I can tell this is what is desired in the code, I realize that was not your verbiage but I think its worth fixing while we are here.

Reply:
I can do that